Makefile

```diff
@@ -30,11 +28,11 @@ vet:
 		exit 1; \
 	fi
 
-.PHONY: fmt
 fmt:
 	@echo "==> Fixing source code with gofmt..."
-	gofmt -s -w ./$(PKG_NAME)
+	gofmt -s -w $(PKG_NAME)
 
-.PHONY: fmtcheck
 fmtcheck:
 	@sh -c "'$(CURDIR)/scripts/gofmtcheck.sh'"
+
+.PHONY: build install test testacc vet fmt fmtcheck
```

Comment:
please don't change that to one `.PHONY` list and follow the rule

> Add each phony target as a prerequisite of .PHONY immediately before the target declaration, rather than listing all the phony targets in a single place.

it's easier to maintain

Reply:
It looks a bit clearer this way don't you think?
Also some other providers do same
https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/GNUmakefile#L61
https://github.com/terraform-providers/terraform-provider-digitalocean/blob/master/GNUmakefile#L61

If you insist I can roll this back though.

Reply:
I would argue if this is clearer. But ok, let's leave like that since other providers does the same as you mentioned. At least that Makefile is not so big